IO/IODispatcher.cpp

```diff
@@ -591,7 +685,8 @@ void TrajectoriesVTK::WriteHeader(long nPeds, double fps, Building * building, i
 void TrajectoriesVTK::WriteGeometry(Building * building)
 {
     (void) building; // avoid warning
-    Log->Write("WARNING:\t Creating NavMesh is deprecated. Please have a look at old files on git "
+    Log->Write("WARNING:\t Creating NavMesh is deprecated. Please have a look at "
+               "old files on git "
```

Comment:
Those line breaks seem to be strange. Maybe put everything on a single line and let clang format split it again, hopefully just one time ...

Reply:
Will be removed soon in an other PR soon. I just need to merge this PR and the one in the pipeline.